ChustaSoft.GamersPlatformUtils.Domain.Steam/Implementations/SteamBusiness.cs

```diff
@@ -8,9 +7,9 @@
 using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 
-namespace ChustaSoft.GamersPlatformUtils.Domain.Implementations
+namespace ChustaSoft.GamersPlatformUtils.Domain
 {
-    public class SteamBusiness : PlatformBase, IAnalyzer, ICleaner
+    public class SteamBusiness : PlatformBase, ISteamBusiness, IAnalyzer, ICleaner
```

Comment:
As ISteamBusiness is now inheriting from IAnalyzer and ICleaner, should we remove IAnalyzer and ICleaner from here?

Reply:
Same above